routers/repo/projects.go

```diff
@@ -286,10 +286,24 @@ func ViewProject(ctx *context.Context) {
 	allBoards := models.ProjectBoardList{uncategorizedBoard}
 	allBoards = append(allBoards, boards...)
 
-	if ctx.Data["Issues"], err = allBoards.LoadIssues(); err != nil {
+	issueList, err := allBoards.LoadIssues()
+	if err != nil {
 		ctx.ServerError("LoadIssuesOfBoards", err)
 		return
 	}
+	ctx.Data["Issues"] = issueList
+
+	linkedPrs := make(map[int64][]*models.Issue)
+	for _, issue := range issueList {
+		for _, comment := range issue.Comments {
+			if comment.RefIssueID != 0 && comment.RefIsPull {
+				if pr, err := models.GetIssueWithAttrsByID(comment.RefIssueID); err == nil {
```

Comment:
This is not efficient? Maybe we can get all IDs at first and then load them from databases batchly?

Reply:
You are certainly right, I totally missed the method to gather all of them in batch. It is implemented now. Unfortunately this made the code look more complex that perhaps it should, if you have any suggestion to make it cleaner please tell me.

Reply:
I have updated this again to use `models.Issues()` rather than `GetIssuesByIDs`, as it seems that the former already populates the attributes, resulting in a shorter patch.